guide/source/2.6-migration.md

```diff
@@ -0,0 +1,87 @@
+  - waitForStepDownOnNonCommandShutdown=false is not needed anymore when spawning the mongodb process
+  - _synchronousCursor._dbCursor.operation is not an option anymore in the raw cursor from nodejs mongodb driver. If you want to access the options, use _synchronousCursor._dbCursor.(GETTERS) - for example, _synchronousCursor._dbCursor.readPreference.
+  
+Below we describe a few common cases in this migration:
+
+#### 1) Same version of MongoDB server
+
+If you are not changing your MongoDB server version you don't need to change anything in your code, even if you are using `rawCollection` results, but as we did many changes on how Meteor interact with MongoDB in order to be compatible with the new driver we recommend that you test your application carefully before releasing to production with this Meteor version.
+
+We've made many tests in real applications and also in our automatic tests suite, we believe we've fixed all the issues that we found along the way but Meteor interaction with MongoDB is so broad and open that maybe you have different use cases that could lead to different issues.
+
+Again, we are not aware of any issues that were introduced by these changes, but it's important that you check your app behavior, especially if you have places where you believe you are not using MongoDB in a traditional way.
+
+#### 2) Migrating from MongoDB 4.x to MongoDB 5.x
+
+As we have made many changes to Meteor core packages in this version we recommend the following steps to migrate:
+- Upgrade your app to use Meteor 2.6 (meteor update --release 2.6) in a branch;
+- Create a staging environment with MongoDB 5.x and your app environment using the branch created in the previous step;
+  - If you are using MongoDB Atlas, in MongoDB Atlas we were not able to migrate to a free MongoDB 5.x instance, so we had to migrate to a paid cluster in order to test the app properly, maybe this will change after February 2022; 
+- Set up your staging database with MongoDB 5.x and restore your production data there, or populate this database in a way that you can reproduce the same cases as if you were in production;
+- Run your app pointing your MONGO_URL to this new database, that is running MongoDB 5.x;
+- Run your end-to-end tests in this environment. If you don't have a robust end-to-end test we recommend that you test your app manually to make sure everything is working properly.
+- Once you have a stable end-to-end test (or manual test), you can consider that you are ready to run using MongoDB 5.x, so you can consider it as any other database version migration.
+
+We are not aware of any issues that were introduced by the necessary changes to support MongoDB, but it's important that you check your app behavior, especially if you have places where you believe you are not using MongoDB in a traditional way.
```

Comment:
This sentence appears to be duplicated, almost the same was said in line 46.
I'm not sure, but maybe that was intentional to reinforce the importance of testing.

Reply:
Each project will be in a different case (1 or 2) so people should read one or the other and not both. Got it?